modules/browsemap/src/main/java/com/sygic/modules/browsemap/BrowseMapFragment.kt

```diff
@@ -31,37 +40,81 @@ class BrowseMapFragment : MapFragmentWrapper() {
     private lateinit var positionLockFabViewModel: PositionLockFabViewModel
     private lateinit var zoomControlsViewModel: ZoomControlsViewModel
 
+    /**
+     * A *[MapSelectionMode]* defines the three available [BrowseMapFragment] selection modes.
+     *
+     * [MapSelectionMode.FULL] -> Enable an unlimited selection mode on the map, every click will be processed.
+     *
+     * [MapSelectionMode.MARKERS_ONLY] (default) -> Limit selection mode to the custom [MapMarker]s only, see [addMapMarker] or [addMapMarkers] methods.
+     *
+     * [MapSelectionMode.NONE] -> Disable any selection on the map.
+     */
     @MapSelectionMode
     var mapSelectionMode: Int
         get() = browseMapFragmentViewModel.mapSelectionMode
         set(value) {
             browseMapFragmentViewModel.mapSelectionMode = value
         }
 
+    /**
+     * A *[compassEnabled]* modifies the [CompassView] visibility.
+     *
+     * @param [Boolean] true to enable the [CompassView], false otherwise.
```

Comment:
ten odkaz na boolean by som tam nedaval to je dost zrejme zo zapisu metody

```suggestion
     * @param true to enable the [CompassView], false otherwise.
```

Reply:
aj ked mozno bez toho mena tam ten Boolean nie je zly napad, neviem takto z kodu sa rozhodnut

Reply:
oki, tak to zatím nechám tak, jak jsme se domluvili